src/filter.jl

```diff
@@ -217,9 +218,18 @@ function _backwardfilter(filter::LyapunovFilter,k::SDEKernel, (c, ν, P); apply_
   end
 
   sol = solLyapunov(k, Σ, ν, P, c)
-  soldis = hcat(sol.(_ts)...)
+
```

Comment:
the new version works for an `ArrayPartition` as well. (The `hcat()` was harcoded for `sol(t)` returning a vector.)